docs/sdk-nft/README.md

```diff
@@ -62,33 +62,151 @@ nft.jsonData;
 // }
 ```
 
-#### NFTFactory
+### NFTImageFactory
 
-Given a NFT721Manager, a StorageType (StorageIpfs, StorageAws, StorageHttp) and a NFT_METADATA_FORMAT, it creates a NFTFactory instance to interact with NFTs
+- createNFT:
+
+Static method that, given a Resource (which returns an image) and a tokenId, it returns an NFT instance.
+
+```javascript
+// commonJS
+const { NFTImageFactory } = require('@rgc/sdk');
+// TS
+import { NFTImageFactory } from '@rgc/sdk';
+
+NFTImageFactory.createNFT(RESOURCE, "aTokenId");
```

Comment:
Should we add the rest of the code for the user to copy, paste and run? (function with async and await but obviously without the nft data)

Just an example:
```
const m = async () => {
    const nft = await NFTImageFactory.createNFT(RESOURCE, "aTokenId");
}
m()
```

And the same with the following examples

Reply:
You are right. I'm just going to add de `await`, like the rest of the docs